### ISSUE TITLE
block rotations from old schematics are messed up

### ISSUE THREAD

**Issue by ghost** [IntellectualSites/FastAsyncWorldEdit#2951]
### Server Implementation

Paper

### Server Version

1.21

### Describe the bug

Block rotations from old schematics are incorrect

### To Reproduce

[manoir10756380.zip](https://github.com/user-attachments/files/17447696/manoir10756380.zip)
load manoir.schematic, load it, wrong rotations, the rotations are definitely saved correctly inside the file, I see this with many old schematics

### Expected behaviour

Should load correctly

### Screenshots / Videos

![image](https://github.com/user-attachments/assets/b07f5775-2868-420a-9fdb-53dd9f5aae11)

![image](https://github.com/user-attachments/assets/15eb4f22-c785-42a5-bb0d-e33588a222f1)

### Error log (if applicable)

No console errors

### Fawe Debugpaste

N/A

### Fawe Version

FastAsyncWorldEdit-Bukkit-2.11.3-SNAPSHOT-937.jar

### Checklist

- [X] I have included a Fawe debugpaste.
- [X] I am using the newest build from https://ci.athion.net/job/FastAsyncWorldEdit/ and the issue still persists.

**Comment by dordsor21** [comment]
I do not believe this data is stored in the schematic - would be "fixed" by #2746
